launchers/launcher.go

```diff
@@ -110,7 +110,7 @@ func LaunchModules(ctx context.Context, modules config.ServerType) (context.Canc
 
 	if modules.IsEnabled(config.DirectorType) {
 
-		viper.Set("Director.DefaultResponse", "cache")
+		viper.Set("Director.DefaultResponse", param.Director_DefaultResponse.GetString())
```

Comment:
What's the intension here to set the value by the value itself?

Reply:
Good point 😆 I noticed that because we had `viper.Set("Director.DefaultResponse", "cache")`, changing the default response in my config had no effect. What I didn't stop to think about was that I could just delete this line all together.

Reply:
Actually, on second thought... I should probably set this as a default with viper. I don't know that everything that reaches this code will necessarily ingest our `config/resources/defaults.yaml`, especially various tests.

Reply:
I think tests should be fine, the ones that could be affected by this should be director-specific tests and e2e tests, which should either set this value explicitly, or call InitConfig to read `default.yaml`.

Reply:
I made the change before I saw this comment. If you think I should remove the line entirely, I can do that too.